### PR TITLE
Adobe Analytics Integration

### PR DESCRIPTION
**Pull request by robertsevernsentient**
Out of the box adobe analytics integration

AP-207

### CODE REVIEW

File: packages/adobe/adobe-analytics/src/adapter.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    getHandler() {
+        return this.customEventHandler || this.getAnalytics();
+    }
+
+    checkAnalyticsProviders() {
+        // @ts-ignore
+        if (window.google_tag_manager || window.gtag) {
+            console.log('Evolv: Analytics integration detected Google Tag Manager - please use \'Evolv.GtagClient()\'');
```
Comment (mattstrom) [inline packages/adobe/adobe-analytics/src/adapter.ts:43]: Is this correct?

File: packages/adobe/adobe-analytics/src/adapter.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        }
+    }
+
+    getDimensionForAdobe(dimension: DimensionsValue) {
+        return (dimension.type || 'eVar') + dimension.key;
+    }
+
+    sendMetrics(type: string, event: any) {
+        var augmentedSid = this.getAugmentedSid();
```
Comment (mattstrom) [inline packages/adobe/adobe-analytics/src/adapter.ts:52]: `var`? Intentional?

File: packages/adobe/adobe-analytics/src/adapter.ts (hunk 1 above)
Comment (robertsevernsentient) [reply packages/adobe/adobe-analytics/src/adapter.ts:43]: absolutely not

File: packages/adobe/adobe-analytics/src/adapter.ts (hunk 2 above)
Comment (robertsevernsentient) [reply packages/adobe/adobe-analytics/src/adapter.ts]: changed to let

**Pull request merged**